Docs/source/usage/workflows/python_extend.rst

```diff
@@ -134,9 +134,12 @@ This example accesses the :math:`E_x(x,y,z)` field at level 0 after every time s
        warpx = sim.extension.warpx
 
        # data access
-       E_x_mf = warpx.multifab(f"Efield_fp[x][level=0]")
+       #   vector field E, component x, on the fine patch of MR level 0
+       E_x_mf = warpx.multifab(f"Efield_fp", dir=0, level=0)
```

Comment:
A quick comment - the `f` prefix is not needed on the strings.

Reply:
True! not sure how this slipped in, maybe a more general example.